src/fixtures.js

```diff
@@ -35,5 +36,7 @@ function syncXhr (filePath) {
     }
 
     return Buffer.from(res)
+  } else {
+    throw new Error('Could not get the Fixture')
```

Comment:
would be nice to say what fixture failed to load

Reply:
Good point, done :)